src/overlays/actors/ovl_En_fHG/z_en_fhg.c

```diff
@@ -27,29 +61,643 @@ const ActorInit En_fHG_InitVars = {
     (ActorFunc)EnfHG_Update,
     (ActorFunc)EnfHG_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_fHG/EnfHG_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_fHG/EnfHG_Destroy.s")
+static EnfHGPainting sPaintings[6] = { { { 0.0f, 60.0f, -315.0f }, 0x0000 },   { { -260.0f, 60.0f, -145.0f }, 0x2AAA },
+                                       { { -260.0f, 60.0f, 165.0f }, 0x5554 }, { { 0.0f, 60.0f, 315.0f }, 0x7FFE },
+                                       { { 260.0f, 60.0f, 155.0f }, 0xAAA8 },  { { 260.0f, 60.0f, -155.0f }, 0xD552 } };
+
+static InitChainEntry sInitChain[2] = {
+    ICHAIN_S8(naviEnemyId, 26, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 1200, ICHAIN_STOP),
+};
+
+void EnfHG_Init(Actor* thisx, GlobalContext* globalCtx2) {
+    GlobalContext* globalCtx = globalCtx2;
+    EnfHG* this = THIS;
+
+    Actor_ProcessInitChain(&this->actor, sInitChain);
+    Flags_SetSwitch(globalCtx, 0x14);
+    Actor_SetScale(&this->actor, 0.011499999f);
+    this->actor.gravity = -3.5f;
+    ActorShape_Init(&this->actor.shape, -2600.0f, NULL, 20.0f);
+    this->actor.speedXZ = 0.0f;
+    this->actor.focus.pos = this->actor.world.pos;
+    this->actor.focus.pos.y += 70.0f;
+    func_800A663C(globalCtx, &this->skin, &gPhantomHorseSkel, &gPhantomHorseAnim_00B4C8);
+
+    if (this->actor.params >= 10) {
```

Comment:
believe you have an enum for this?

Reply:
I don't, as it's not an enum. I should have a #define for that constant, though.